src/storekit.cs

```diff
@@ -469,5 +469,60 @@ interface SKCloudServiceController {
 		[Field ("SKCloudServiceCapabilitiesDidChangeNotification")]
 		NSString CloudServiceCapabilitiesDidChangeNotification { get; }
 	}
+
+	[NoTV, iOS (10,1)]
+	[BaseType (typeof(UIViewController))]
+	interface SKCloudServiceSetupViewController
+	{
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		ISKCloudServiceSetupViewControllerDelegate Delegate { get; set; }
+
+		[Async]
+		[Export ("loadWithOptions:completionHandler:")]
+		void Load (NSDictionary options, [NullAllowed] Action<bool, NSError> completionHandler);
+
+		[Wrap ("Load (options == null ? null : options.Dictionary, completionHandler)")]
+		void Load (SKCloudServiceSetupOptions options, Action<bool, NSError> completionHandler);
```

Comment:
I'm fairly sure this won't wrap the async version, so another one is needed for `LoadAsync`.